torchx/schedulers/ray/ray_driver.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import json
+import dataclasses
+import ray
+from typing import Any, Dict, Iterable, List, Optional, Set, Type
+import subprocess
+from ray_common import RayActor
+
+@ray.remote
+class CommandActor:
+    def __init__(self, command, env):
+        self.command = command
+        self.env = env
+
+    def run_command(self):
+        return exec(object = self.command, globals = self.env)
```

Comment:
I don't think you can pass in arbitrary commands to `exec`- it can only take in Python statements.

So instead of passing in a `command` to the actor, we can change this to pass in the name of the train script to run. And then you would do `exec(open(self.train_script))`.